src/renderer/entities/walletConnect/model/wallet-connect-model.ts

```diff
@@ -33,17 +33,18 @@ type ConnectResult = {
 };
 
 type SessionTopicUpdateProps = {
```

Comment:
I'd rather name it `SessionTopicParams`, `Props` is a react-world naming
effector is framework agnostic 😉